### ISSUE TITLE
Project with path ':@react-native-firebase_app' could not be found in project ':react-native-firebase_auth'

### ISSUE THREAD

**Issue by AnthonyMwangi** [invertase/react-native-firebase#3216]
## Issue

I was getting this error after adding Authentication to an existing project and could not get any solutions or even references to the same issue anywhere, 

There were similar issues but relating to analytics and that's where I found a solution. so I'm raising this issue to assist anyone who may encounter it and save them the 2 days it took me to finally solve it or hopefully get a better solution since this one is kind of a workaround.

### **Solution**
It basically involves creating a new project from the[ Quick Start Template ](https://invertase.io/oss/react-native-firebase/quick-start/new-project#initializing-the-project)  and then transferring the existing project files and installing the [Authentication](https://invertase.io/oss/react-native-firebase/v6/auth#authentication) Module or any other dependencies to your project.

**Step 1: (Command to Create New Project)**
`npx @react-native-community/cli init --template=@react-native-firebase/template <new_project_name>`

**Step 2: (Install Authentication Module)**
`yarn add @react-native-firebase/auth` 

**Step 3: (Install Other Dependencies)**
Copy the other dependencies from your _package.json_ then run the  `yarn install` or `npm install` to install and auto-link them if applicable.

**_---_**
You should now be able to build your app. 

---

### Javascript

<details><summary>Click To Expand</summary>
<p>

#### `package.json`:

```json
# N/A
```

#### `firebase.json` for react-native-firebase v6:

```json
# N/A
```

### iOS

<details><summary>Click To Expand</summary>
<p>

#### `ios/Podfile`:

- [ ] I'm not using Pods
- [x] I'm using Pods and my Podfile looks like:

```ruby
# N/A
```

#### `AppDelegate.m`:

```objc
// N/A
```

</p>
</details>

---

### Android

<details><summary>Click To Expand</summary>
<p>

#### Have you converted to AndroidX?

- [ ] my application is an AndroidX application?
- [ ] I am using `android/gradle.settings` `jetifier=true` for Android compatibility?
- [ ] I am using the NPM package `jetifier` for react-native compatibility?

#### `android/build.gradle`:

```groovy
// N/A
```

#### `android/app/build.gradle`:

```groovy
// N/A
```

#### `android/settings.gradle`:

```groovy
// N/A
```

#### `MainApplication.java`:

```java
// N/A
```

#### `AndroidManifest.xml`:

```xml
<!-- N/A -->
```

</p>
</details>

---

## Environment

<details><summary>Click To Expand</summary>
<p>

**`react-native info` output:**

```
 OUTPUT GOES HERE
```

- **Platform that you're experiencing the issue on**:
  - [ ] iOS
  - [ ] Android
  - [ ] **iOS** but have not tested behavior on Android
  - [ ] **Android** but have not tested behavior on iOS
  - [ ] Both
- **`react-native-firebase` version you're using that has this issue:**
  - `e.g. 5.4.3`
- **`Firebase` module(s) you're using that has the issue:**
  - `e.g. Instance ID`
- **Are you using `TypeScript`?**
  - `Y/N` & `VERSION`
  
</p>
</details>

---

Think `react-native-firebase` is great? Please consider supporting all of the project maintainers and contributors by donating via our [Open Collective](https://opencollective.com/react-native-firebase/donate) where all contributors can submit expenses. [[Learn More]](https://invertase.io/oss/react-native-firebase/contributing/donations-expenses)

- 👉 Check out [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter for updates on the library.

**Comment by mikehardy** [comment]
This was discussed with a solution discovered and publicized yesterday as #3206 and fixed in PR #3215 which should be released shortly

In the meantime the workaround of `yarn add @react-native-community/cli-platform-android@3.0.3` is valid

**Comment by Salakar** [comment]
Released in 6.3.4
